libraries/AP_Mount/AP_Mount_SToRM32_serial.cpp

```diff
@@ -272,7 +272,6 @@ void AP_Mount_SToRM32_serial::parse_reply() {
         case ReplyType_ACK:
             crc = crc_calculate(&_buffer[1],
                                 sizeof(SToRM32_reply_ack_struct) - 3);
-            crc_ok = crc == _buffer.ack.crc;
```

Comment:
Can't we remove the whole thing? The `crc` gets unused now. It looks like it should be:

```
case ReplyType_ACK:
default:
    break;
```